### PR TITLE
feat: refactor chart to use native elements, remove Highcharts

### PR DESCRIPTION
**Pull request by jessicamcinchak**
changes: 
- now just grid of divs styled with tailwind (3 equal width columns x N rows based on number of patterns/patternClass rows) - still pretty responsive I think :crossed_fingers: 
- ability to chart by individual terms or rollup to pattern class
- ability to show/hide labels (shifts to 2 column grid)

### CODE REVIEW

File: components/Chart.tsx [hunk 1]
```diff
@@ -1,24 +1,36 @@
...
 const Chart = (props: Props) => {
-  const { showLabels, terms = [], patterns = [], patternClasses = [] } = props
+  const { rollupToPatternClass, showLabels, terms = [], patterns = [], patternClasses = [] } = props;
+
+  // Format the list of individual terms that apply to this entry
+  let formattedTerms = _(terms)
+    .map((term: any) => ({
+      pattern: _.find(patterns, ['_id', term.pattern._ref]),
+      patternName: _.find(patterns, ['_id', term.pattern._ref])?.name,
+      type: term.rightsIntensity > 0 ? "Right" : "Obligation", // because pattern.type is not consistently populated
```
Comment (DafyddLlyr) [inline components/Chart.tsx:21]: This is likely due to the staging / production dataset issues that I'll sort today and then take a close look 👍 

The other issue that would be in play here is that there are some null values for intensity meaning we cant derive a type - I'll see how many there are and see if I can get Niamh to prioritise populating these.

File: components/Chart.tsx [hunk 2]
```diff
@@ -29,144 +41,110 @@ const Chart = (props: Props) => {
...
+  const textColorClasses: any = {
+    "Rent": "text-rent",
+    "Transfer": "text-transfer",
+    "Administration": "text-administration",
+    "Eligibility": "text-eligibility",
+    "Security of tenure": "text-security",
+    "Develop": "text-develop",
+    "Stewardship": "text-stewardship",
+    "Use": "text-gray-400",
+  };
```
Comment (DafyddLlyr) [inline components/Chart.tsx:95]: We're going to need this elsewhere also - can it be in a style utils file or similar?

**Review by DafyddLlyr — APPROVED** [review]
Looking great 👌

File: components/Chart.tsx (hunk 1 above)
Comment (jessicamcinchak) [reply components/Chart.tsx:21]: we could also be a bit smarter here and do something like: `term.type || term.rightsIntensity > 0 ? "Right" : "Obligation"`

i think the side effect of not being able to derive a type is simply not rendering that row on the chart right now - so not ideal, but not breaking!

File: components/Chart.tsx (hunk 2 above)
Comment (jessicamcinchak) [reply components/Chart.tsx:95]: Yep for sure, will make this change now before merging :+1:

Comment (jessicamcinchak) [reply components/Chart.tsx]: So I attempted to import these consts from `'lib/utils.ts`, but that actually raised the same build-time error as I was getting when attempting to template them - fiddled with for awhile, but ultimately leaving in this file for now!

Annoyingly I suspect we may just have to re-define these in each file where we want to use them.

**Pull request merged**